setup.cfg

```diff
@@ -26,7 +26,7 @@ project_urls =
 packages = find:
 install_requires =
     packaging
-    pep517
+    pep517>=0.9
```

Comment:
Well, this a dangerous form. The minimum version required is `0.9.1`. There hasn't been released a `0.9.0`, so this works, but is there any reason why you don't just fully specify the exact minimum version that's guaranteed not to break? Otherwise, an erroneous 0.9.0 release can break us.

Reply:
There has, it was just not released to PyPI. People who get releases from Github may be using it. The fix was introduced in 0.9.0, not 0.9.1.